ls1intum/Artemis
# `Programming exercises`: Fix missing grading instructions after import

### Checklist
#### General


#### Server

### Motivation and Context
When importing an exercise the grading instructions are not copied from the source exercise.

### Description
Makes sure the grading instructions are loaded and copied to the new exercise.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Programming Exercise with manual assessment

1. Edit the exercise and add a few manual grading criteria/instructions if they are not present already.
2. Import this programming exercise.
3. After import, the new programming exercise should have the same grading criteria/instructions.

### Testserver States
> [!NOTE]
> These badges show the state of the test servers.
> Green = Currently available, Red = Currently locked

![](https://byob.yarr.is/ls1intum/Artemis/artemis-test1)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test2)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test3)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test4)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test5)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test6)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test9)

### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests

### Test Coverage
unchanged, only query changed without surrounding production code. Added new tests to make sure that it works as intended.
